toolchain/lower/file_context.cpp

```diff
@@ -374,16 +348,22 @@ auto FileContext::BuildFunctionDefinition(SemIR::FunctionId function_id)
     function_lowering.SetLocal(param_id, param_value);
   };
 
-  // The subset of calling_convention_param_id that is in sequential order.
-  llvm::ArrayRef<SemIR::InstId> sequential_param_ids =
-      calling_convention_param_ids;
-
-  // The LLVM calling convention has the return slot first rather than last.
-  if (has_return_slot) {
-    lower_param(calling_convention_param_ids.back());
-
-    sequential_param_ids = sequential_param_ids.drop_back();
+  // The subset of call_param_ids that is in sequential order.
+  llvm::ArrayRef<SemIR::InstId> sequential_param_ids;
```

Comment:
In the new setup, isn't `call_param_ids` a sequential block? Could you explain what "sequential order" means in this context, or maybe the naming should be updated?

Reply:
I've rephrased the comment: these are the parameters that are already in the order that the LLVM calling convention expects, so we can just lower them in a loop. I'm having trouble coming up with a better name to capture that idea, but I'm open to suggestions.